package.json

```diff
@@ -26,6 +26,9 @@
   "dependencies": {
     "@ant-design/icons": "^4.7.0",
     "@educandu/node-jsx-loader": "^2.2.0",
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
Could you please remove these 3 libraries? As far as I can see, you're not using them anymore, right? 🤔